static/routes.config.php

```diff
@@ -383,6 +383,7 @@
 		'/hidden'                     => [Module\Contact::class,           [R::GET]],
 		'/ignored'                    => [Module\Contact::class,           [R::GET]],
 		'/hovercard'                  => [Module\Contact\Hovercard::class, [R::GET]],
+		'/follow[/{url}]'       	  => [Module\Contact\Follow::class,    [R::GET, R::POST]],
```

Comment:
Watch out for tab characters, they should only be used for the leftmost indentation.
```suggestion
		'/follow[/{url}]'             => [Module\Contact\Follow::class,    [R::GET, R::POST]],
```